src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -313,6 +335,11 @@ public DataAccessRequest createDraftDarFromCanceledCollection(User user, DarColl
             now,
             newData
         );
+
+        // Update the Joins table *dar_dataset
+        dataAccessRequestDAO.deleteDARDatasetRelationByReferenceIds(canceledReferenceIds);
```

Comment:
I'm pretty sure we don't want to do this. Even though a canceled DAR is canceled, that doesn't mean it should not be associated to the dataset that the user requested it to be associated to. We do want to sync the new DAR, but that's really all we want to do here.